lib/srv/desktop/rdp/rdpclient/src/lib.rs

```diff
@@ -132,7 +132,7 @@ impl Client {
             Some(c) => Ok(c),
             None => {
                 error!("invalid Rust client pointer");
-                Err(CGOErrCode::ErrCodeFailure)
+                Err(CGOErrCode::ErrCodeClientPtr)
```

Comment:
My question is why are we adding this? I'm guessing this is returned to the Go side, are we just wanting it to be logged there? Does this indicate a bug in our code we're trying to track down or something out of our control that we expect to happen sometimes? Should we add extra logging or tracing in either case, if we have anything meaningful to log?

Reply:
Noted (though auto merge was on so this already went through).

This error is unlikely to occur unless we introduce a programming error with sharing handles between Go/Rust. I made a separate failure code for this just to help troubleshooting (in the off chance this does happen, we'll at least be able to tell this apart from a more general and expected failure)